6.6.4: Add release notes (#3590)
sensu/sensu-docs: content/sensu-go/6.6/release-notes.md

---

## 6.6.4 release notes

> **January 13, 2022** &mdash; The latest release of Sensu Go, version 6.6.3, is now available for download.

Reviewer: ```suggestion

**January 13, 2022** &mdash; The latest release of Sensu Go, version 6.6.3, is now available for download.
```

> Sensu Go 6.6.4 includes a number of bug fixes, security improvements and the addition of the new metric `sensu_go_event_metric_points_processed`. Fixes in this patch release will help prevent the backend from crashing when PostgreSQL is taken offline and will prevent backend entity rows from filling up the entities table. In addition, fixes will further secure the web interface.

Reviewer: ```suggestion
Sensu Go 6.6.4 includes a number of bug fixes, security improvements, and a new metric, `sensu_go_event_metric_points_processed`.
Fixes in this patch release will help prevent backend crashes when PostgreSQL is taken offline and keep backend entity rows from filling up the entities table.
The 6.6.4 patch release also includes several improvements to further secure the web UI.
```

Read the [upgrade guide][1] to upgrade Sensu to version 6.6.4.

**IMPROVEMENTS**

> - ([Commercial feature][259]) In the web UI, added [X-Frame-Options][263] header to tell browsers the web application cannot be loaded within an iframe; loaded in this manner, it can be susceptible to tailored click jacking attacks. 

Reviewer: ```suggestion
- ([Commercial feature][259]) In the web UI, added the [X-Frame-Options][263] header to tell browsers the web application cannot be loaded within an iframe to prevent tailored click-jacking attacks. 
```

---

## 6.6.4 release notes

> **January 13, 2022** &mdash; The latest release of Sensu Go, version 6.6.4, is now available for download.

Reviewer: The date may need a further update if the release doesn't go on Tuesday.

> **January 13, 2022** &mdash; The latest release of Sensu Go, version 6.6.4, is now available for download.

Reviewer: ```suggestion
**January 26, 2022** &mdash; The latest release of Sensu Go, version 6.6.4, is now available for download.
```

---

## 6.6.4 release notes

> **January 13, 2022** &mdash; The latest release of Sensu Go, version 6.6.3, is now available for download.

Reviewer: ```suggestion
**January 13, 2022** &mdash; The latest release of Sensu Go, version 6.6.4, is now available for download.
```

Sensu Go 6.6.4 includes a number of bug fixes, security improvements, and a new metric, `sensu_go_event_metric_points_processed`.
Fixes in this patch release will help prevent backend crashes when PostgreSQL is taken offline and keep backend entity rows from filling up the entities table.
The 6.6.4 patch release also includes several improvements to further secure the web UI.

Read the [upgrade guide][1] to upgrade Sensu to version 6.6.4.

**IMPROVEMENTS**
- ([Commercial feature][259]) In the web UI, added the [X-Frame-Options][263] header to tell browsers the web application cannot be loaded within an iframe to prevent tailored click-jacking attacks. 

> - ([Commercial feature][259]) In the web UI, added [HSTS header][264] if TLS has been configured; ensures the browser loads the application and its requisite assets with a secure connection.

Reviewer: ```suggestion
- ([Commercial feature][259]) In the web UI, added the [HSTS header][264] if TLS has been configured to ensure that the browser loads the application and its requisite assets with a secure connection.
```

> - ([Commercial feature][259]) In the web UI, added [X-Content-Type-Options][265]: nosniff header to have browsers respect the given Content-Type header when loading content referenced by a script tag. 

Reviewer: ```suggestion
- ([Commercial feature][259]) In the web UI, added the [X-Content-Type-Options][265] nosniff header so that browsers respect the given Content-Type header when loading content referenced by a script tag. 
```

> - Add sensu_go_event_metric_points_processed counter metric and included it in tessen reporting.

Reviewer: ```suggestion
- Added the `sensu_go_event_metric_points_processed` counter metric and included it in Tessen reporting.
```

**FIXES**

> - ([Commercial feature][259]) Fixed bugs in BSM and round robin scheduling. When postgres round robin scheduling is enabled, users will no longer see missed check executions.

Reviewer: ```suggestion
- ([Commercial feature][259]) Fixed bugs in business service monitoring (BSM) and round robin scheduling to prevent missed check executions when PostgreSQL round robin scheduling is enabled.
```

> - ([Commercial feature][259]) Fixed a bug where sensu-backend would crash if postgresql was taken offline and started up again.

Reviewer: ```suggestion
- ([Commercial feature][259]) Fixed a bug that could cause sensu-backend to crash if PostgreSQL was taken offline and restarted.
```

> - ([Commercial feature][259]) Fixed a bug where ephemeral backend entity rows would fill up the entities table in postgresql.

Reviewer: ```suggestion
- ([Commercial feature][259]) Fixed a bug that could cause ephemeral backend entity rows to fill up the entities table in PostgreSQL.
```

> - ([Commercial feature][259]) BSM event selectors can no longer select events outside the ServiceComponent namespace.

Reviewer: ```suggestion
- ([Commercial feature][259]) BSM event selectors can no longer select events outside the service component namespace.
```

> - Referencing a non-existent handler in a pipeline no longer results in a crash.

Reviewer: ```suggestion
- Fixed a bug that could cause the backend to crash if a pipeline references a non-existent handler.
```

> 

Reviewer: ```suggestion
```
